bindings/node/package.json

```diff
@@ -52,6 +52,7 @@
     "jsdoc-to-markdown": "^7.1.1",
     "mocha": "^9.2.0",
     "mongodb": "^4.3.1",
+    "mongodb-client-encryption": "file:.",
```

Comment:
This is so the driver package can pick up this one, so that the auto-encryption-enabled client in the spec tests works.